### PR TITLE
contrib/test/ci: set cgroup_config for fedora

### PR DESCRIPTION
**Pull request by sohankunkerkar**
This is needed to run the kubernetes cluster without any hiccups on Fedora 36

Signed-off-by: Sohan Kunkerkar <sohank2602@gmail.com>

#### What type of PR is this?

/kind ci

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: contrib/test/ci/build/kubernetes.yml [hunk 1]
```diff
@@ -82,6 +82,15 @@
       ./hack/local-up-cluster.sh
     mode: "u=rwx,g=rwx,o=x"
 
+- name: Set CGROUP_CONFIG for fedora
+  become: yes
+  lineinfile:
+    dest: /usr/local/bin/createcluster.sh
```
Comment (mrunalp) [inline contrib/test/ci/build/kubernetes.yml:88]: Why can't we put this inside the createcluster.sh script?

Comment (sohankunkerkar) [reply contrib/test/ci/build/kubernetes.yml:88]: I don't think we're hitting this issue for other variants, so tried the safer approach.

**Comment by haircommander** [comment]
@sohankunkerkar do we still need this?

**Comment by sohankunkerkar** [comment]
I don't think we need this anymore

**Pull request closed without merging**